drivers/wdm/audio/legacy/wdmaud/sup.c

```diff
@@ -389,15 +405,14 @@ GetSysAudioDevicePnpName(
         return STATUS_INSUFFICIENT_RESOURCES;
 
     /* query sysaudio again for the device path */
-    Status = KsSynchronousIoControlDevice(DeviceExtension->FileObject, KernelMode, IOCTL_KS_PROPERTY, (PVOID)&Pin, sizeof(KSPROPERTY) + sizeof(ULONG), (PVOID)*Device, BytesReturned, &BytesReturned);
+    Status = KsSynchronousIoControlDevice(DeviceEntry->FileObject, KernelMode, IOCTL_KS_PROPERTY, (PVOID)&Pin, sizeof(KSPROPERTY) + sizeof(ULONG), (PVOID)*Device, BytesReturned, &BytesReturned);
 
     if (!NT_SUCCESS(Status))
     {
         /* failed */
         FreeItem(*Device);
         return Status;
     }
-
```

Comment:
readd this newline. no need for cosmetic changes here.

Reply:
Fixed (I think).

Reply:
no. it's not fixed. now that line contains unintended spaces that were not there before.

Reply:
How do I fix it?

Reply:
You literally delete the line and press enter, don't add a space or anything.. incredibly simple.

Reply:
What lines do I delete?